src/services/db/GitHubService.ts

```diff
@@ -861,9 +863,9 @@ export default class GitHubService {
     const endpointTemplate = urlTemplate.parse(`{siteName}`)
     const endpoint = endpointTemplate.expand({ siteName })
 
-    // Privatising a repo is restricted to repo admins - an admin token will be inserted in via our axios interceptor
+    // Privatising a repo is restricted to repo admins - our main admin token is used
     const headers = {
-      Authorization: "",
```

Comment:
How come this was previously empty? Comment seems to say this was injected by the axios interceptor 

If we are modifying to inject token here is the interceptor still needed

Reply:
interceptor always exists, and inserts in the token in cases where it doesn't exist (i.e. all email login actions). normally this is just to catch the github flow, where the user's personal github token wold be used (but they would lack perms)

In this case we're using the admin token for both flows, so no change